MacC-GoldenRatio/MacC-GoldenRatio/Sources/Presenter/PageScene/ViewController/PageEditModeViewController.swift

```diff
@@ -208,6 +210,22 @@ class PageEditModeViewController: UIViewController {
 		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.font: UIFont.body, NSAttributedString.Key.foregroundColor:UIColor.black]
     }
 
+    private func setNavigationButtonNotification() {
+        NotificationCenter.default.addObserver(self, selector: #selector(configureNavigationConfirmButtonEnabled(_:)), name: .isNavigationConfirmButtonEnabled, object: nil)
+    }
+    
+    private func deleteNavigationButtonNotification() {
+        NotificationCenter.default.removeObserver(self, name: .isNavigationConfirmButtonEnabled, object: nil)
+    }
+    
+    @objc func configureNavigationConfirmButtonEnabled(_ notification: Notification) {
+        guard let isEnabled = notification.object as? Bool else { return }
+
+        DispatchQueue.main.async {
+            self.navigationItem.rightBarButtonItem?.isEnabled = isEnabled
+        }
```

Comment:
p1: 현재 버튼이 disable되는 동안 버튼에 attribute 적용이 풀려서 폰트 적용이 안되는 것 같습니다! disable 상황에서 폰트 속성 적용하는 코드를 추가해주시면 좋을 것 같습니다!
```suggestion
        DispatchQueue.main.async {
            self.navigationItem.rightBarButtonItem?.isEnabled = isEnabled
            self.navigationItem.rightBarButtonItem?.setTitleTextAttributes([NSAttributedString.Key.font: UIFont.body], for: .disabled)
        }
```

Reply:
오호 놓치고 있던 부분이네요!
commit suggestion 까지.. 감사합니다!!